R/BoxCox.R

```diff
@@ -33,9 +33,14 @@
 #'
 #' # Tidying
 #'
-#' When you [`tidy()`][tidy.recipe()] this step, a tibble with columns
-#' `terms` (the selectors or variables selected) and `value` (the
-#' lambda estimate) is returned.
+#' When you [`tidy()`][tidy.recipe()] this step, a tibble returned with 3
+#' columns `terms`, `value` and `id`:
```

Comment:
```suggestion
#' columns `terms`, `value`, and `id`:
```

I can't find any references in slack messages or the tidyverse design guide—maybe besides the existence of `str_flatten_comma()`?—but I'd like to see oxford commas here. Seems like these would be consistent with much of recipes' documentation currently: https://github.com/search?q=repo%3Atidymodels%2Frecipes%20%22%2C%20and%22&type=code.

Reply:
done in ba6b743f3